open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-client/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/client/OpenLineage.java

```diff
@@ -42,11 +44,14 @@ public OpenLineage(String serverName,
      * @return A subgraph containing all relevant paths, in graphSON format.
      * @throws InvalidParameterException one of the parameters is null or invalid
      */
+  //  public LineageVerticesAndEdges lineage(String userId, GraphName graphName, Scope scope, View view, String guid) throws InvalidParameterException {
     public String lineage(String userId, GraphName graphName, Scope scope, View view, String guid) throws InvalidParameterException {
         String methodName = "lineage";
         String url = "/servers/{0}/open-metadata/open-lineage/users/{1}/lineage/sources/{2}/scopes/{3}/views/{4}/entities/{5}";
         return getRestCall(url, String.class, serverName, userId, graphName.getText(), scope.getValue(), view.getValue(), guid);
```

Comment:
The client should handle exception responses and throw checked exceptions. This can be done in separate PR